includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -60,6 +60,92 @@ public function checkAntiSpoof() {
 
 		return;
 	}
+	
+	/**
+	 * AntiSpoof: update records once a new account has been created.
+	 * TODO: remove after post-registration hooks are fixed
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)